client/src/controllers/scores/scores_controller.js

```diff
@@ -21,12 +21,16 @@ class ScoresController {
 		this.$scope.$on('reload', () => self.load(true))
 		this.Messanger.on('reload', () => self.load(false), true)
 		this.Tournament.teams().then(teams => {
-			this._loading = false
 			this._teamNumberList = []
 			for (var i = 0; i < teams.length; i++) {//Creates list of team numbers.
 				this._teamNumberList.push(teams[i].number)
 			}
 		})
+
+
+		self.search = ''
+		self.match = null
+		this._loading = false

```

Comment:
The point of the `_loading` variable is to be `true` until all neccessary data was loaded for this view. In this view, it occurs when the `teams` have been loaded. Please return the `_loading = false` back into the promise

Reply:
There is no pre-existing promise in this controller. Should I implement one?

Reply:
The teams loading is a promise, and therefore can be chained with a `then` function